src/app_service/service/community/service.nim

```diff
@@ -445,4 +450,46 @@ QtObject:
         return response.result.getStr
     except Exception as e:
       error "Error exporting community", msg = e.msg
-      raise newException(RpcException, "Error exporting community: " & e.msg)
\ No newline at end of file
+      raise newException(RpcException, "Error exporting community: " & e.msg)
+
+  proc acceptRequestToJoinCommunity*(self: Service, requestId: string) =
+    try:
+      let response = status_go.acceptRequestToJoinCommunity(requestId)
```

Comment:
@jrainville since `response` is not used anywhere you can just discard it (in general as you wish, I am ok with this as well), states for this and methods bellow.